gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2775,6 +2789,26 @@ func funcOf(last BlockNode) (BlockNode, *FuncTypeExpr) {
 	}
 }
 
+func findBreakableNode(ns []Node) bool {
+	for _, n := range ns {
+		switch n.(type) {
+		case *ForStmt, *RangeStmt, *SwitchClauseStmt, *SwitchStmt:
```

Comment:
curious do we need `*SwitchStmt` here?

Reply:
I guess that could only happen if we were to execute a self-executing closure... and that got me thinking whether we handled self-executing closures at all, even in the runtime code?

[And yeah, we don't](https://github.com/gnolang/gno/issues/2973).

I removed the SwitchStmt :)